README.md

```diff
@@ -186,3 +186,50 @@ From the _root_ directory of the repository:
 ```shell
 pytest tests/
 ```
+
+Docker Setup:
+This Section deals with setting up of Variation Normalizer via docker. For detailed instructions on variation Normalizer and its developer setup , Please refer to the variation normalizer Home page : https://github.com/cancervariants/variation-normalization/tree/main
```

Comment:
Not necessary to include the last sentence since it will be merged into `main` branch eventually 
```suggestion
This Section deals with setting up Variation Normalizer via Docker
```

Reply:
Lines removed.